spark/v3.2/spark/src/main/java/org/apache/iceberg/spark/source/SparkScanBuilder.java

```diff
@@ -108,8 +108,8 @@ public Filter[] pushFilters(Filter[] filters) {
         expr = SparkFilters.convert(filter);
       } catch (IllegalArgumentException e) {
         // converting to Iceberg Expression failed, so this expression cannot be pushed down
-        LOG.warn("Failed to convert filter to Iceberg expression, skipping push down for this expression: {}",
-            filter, e);
+        LOG.info("Failed to convert filter to Iceberg expression, skipping push down for this expression: {}.{}",
```

Comment:
Nit, could you add one space between the two {}, so that it's not jumbled into one word?

Reply:
Done. Thanks!